utils/build.ps1

```diff
@@ -2293,11 +2293,14 @@ function Write-SDKSettingsPlist([OS] $OS) {
   Write-PList -Settings $SDKSettings -Path "$(Get-SwiftSDK $OS)\SDKSettings.plist"
 
   $SDKSettings = @{
-    CanonicalName = $OS.ToString()
-    DisplayName = $OS.ToString()
-    IsBaseSDK = "NO"
+    CanonicalName = $Platform.ToString()
+    DisplayName = $Platform.ToString()
+    IsBaseSDK = "YES"
     Version = "${ProductVersion}"
     VersionMap = @{}
+    HeaderSearchPaths = @( "usr/include" );
+    LibrarySearchPatsh = @();
+    Toolchains = @( "${ToolchainIdentifier}" );
     DefaultProperties = @{
       PLATFORM_NAME = $OS.ToString()
       DEFAULT_COMPILER = "${ToolchainIdentifier}"
```

Comment:
Note: I don't think DEFAULT_COMPILER does anything at all as far as I can see, you should be able to remove it.